pyseq.py

```diff
@@ -66,9 +66,12 @@
 # regex for matching format directives
 format_re = re.compile(r'%(?P<pad>\d+)?(?P<var>\w+)')
 
+# character to join explicit frame ranges on
+range_join = os.environ.get("PYSEQ_SEP", " ")
```

Comment:
Ah, I see. OK, I changed it to PYSEQ_RANGE_SEP, and I also updated `uncompress` to use `range_join` (it was hardcoded to `" "`).